app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -1832,13 +1840,6 @@ class BrowserTabViewModel @Inject constructor(
 
         omnibarViewState.value = currentOmnibarViewState().copy(
             isEditing = hasFocus,
-            showVoiceSearch = voiceSearchAvailability.shouldShowVoiceSearch(
-                hasFocus = hasFocus,
-                query = query,
-                hasQueryChanged = hasQueryChanged,
-                urlLoaded = url ?: "",
-            ),
-            omnibarText = query,
```

Comment:
This is the line that caused the `rq` pixel to fire. Not having it caused the omnibar text not to be updated when typing a new character with voice search enabled